### PR TITLE
Update ASC intelligent repricing text

### PR DESCRIPTION
**Pull request by hickskenh**
## Purpose of this pull request

This pull request (PR) updates some text for Amazon intelligent repricing rules and adds alert text pertaining to vacation/test mode.

## Affected documentation pages

- https://docs.magento.com/m2/ee/user_guide/sales-channels/amazon/pricing-products.html
- https://docs.magento.com/m2/ee/user_guide/sales-channels/amazon/intelligent-repricing-rules.html
- https://docs.magento.com/m2/ee/user_guide/sales-channels/amazon/floor-price.html
- https://docs.magento.com/m2/ee/user_guide/sales-channels/amazon/optional-ceiling-price.html

## Affected Magento editions

- [X] Open Source
- [X] Commerce
- [X] B2B

## Additional information

request for changes came from channels qa team

### CODE REVIEW

**Comment by hickskenh** [comment]
do not review yet, making changes based on a PR on internal.  will commit changes and update comment

File: src/_includes/amazon-intelligent-pricing-ceiling-configure.md [hunk 1]
```diff
@@ -6,12 +6,12 @@ Select your optional highest price setting in the **Optional Ceiling Price** sec
 
 1. For **Ceiling Price Action**, select an option.
 
-    - **Decrease By**: Choose when you want the defined ceiling price source value to be decreased before listing to Amazon.
+    - **Decrease By**: Choose when you want the defined Ceiling Price Source value to be adjusted down, creating a lower ceiling price for the rule, before listing to Amazon.
```
Comment (leslietilling) [inline src/_includes/amazon-intelligent-pricing-ceiling-configure.md]: I thought we decided on `-` insteand of `:`

File: src/sales-channels/amazon/pricing-products.md [hunk 2]
```diff
@@ -10,6 +10,9 @@ There are two types of pricing rules:
 - [Standard Pricing Rule]({% link sales-channels/amazon/standard-price-rules.md %})
 - [Intelligent Repricing Rule]({% link sales-channels/amazon/intelligent-repricing-rules.md %})
 
+   {:.bs-callout .bs-callout-warning}
+   Intelligent repricing rules do not function properly if the Amazon region is set to `Inactive` status, as it is during onboarding. Your pricing calculations depend on your shipping rates, and your region must be in `Active` status for your shipping rates to sync from Amazon. <br/><br/>To update your region status in your Amazon account, go to Settings > Account Info > Vacation Settings. Refer to [Amazon: Listing Status for Vacations](https://sellercentral.amazon.com/gp/help/help.html?itemID=200135620&amp;language=en_MX&amp;ref=ag_200135620_cont_191){:target="_blank"}.
```
Comment (leslietilling) [inline src/sales-channels/amazon/pricing-products.md]: You don't need `{:target="_blank"}` for links that go outside of the magento primary domain.

File: src/sales-channels/amazon/price-adjustment.md [hunk 3]
```diff
@@ -25,6 +25,6 @@ _Intelligent Repricing Rule: Price Adjustment_
...
+|Price Action|Options:<br>**Decrease By**: Choose when you want the defined price source value to be to be adjusted down, creating a lower price for the rule, before listing to Amazon.<br>**Increase By**: Choose when you want the defined price source value to be adjusted up, creating a higher price for the rule, before listing to Amazon.<br>**Match Competitor Price**: (Intelligent repricing rule only) Choose when you want to change your Amazon listing price to match the lowest competitor price, based on your competitor feedback and variance parameters. When set to `Match Competitor Price`, the **Apply** and **Adjustment Amount** fields are removed.|
```
Comment (erikmarr) [inline src/sales-channels/amazon/price-adjustment.md]:
```suggestion
|Price Action|Options:<br/>**Decrease By**: Choose when you want the defined price source value to be to be adjusted down, creating a lower price for the rule, before listing to Amazon.<br/>**Increase By**: Choose when you want the defined price source value to be adjusted up, creating a higher price for the rule, before listing to Amazon.<br/>**Match Competitor Price**: (Intelligent repricing rule only) Choose when you want to change your Amazon listing price to match the lowest competitor price, based on your competitor feedback and variance parameters. When set to `Match Competitor Price`, the **Apply** and **Adjustment Amount** fields are removed.|
```

Comment (erikmarr) [reply src/sales-channels/amazon/price-adjustment.md]: I don't think this affects how it is rendered, but maybe consider to be consistent with other instances in this PR.

File: src/_includes/amazon-intelligent-pricing-ceiling-configure.md (hunk 1 above)
Comment (hickskenh) [reply src/_includes/amazon-intelligent-pricing-ceiling-configure.md]: for Amazon, everything was done using :   that would be a global change for amazon docs.

File: src/sales-channels/amazon/pricing-products.md (hunk 2 above)
Comment (hickskenh) [reply src/sales-channels/amazon/pricing-products.md]: thank you.  will correct

File: src/sales-channels/amazon/price-adjustment.md (hunk 3 above)
Comment (hickskenh) [reply src/sales-channels/amazon/price-adjustment.md]: thank you.  missed that.  don't recall in the output what happened without the /

File: src/_includes/amazon-intelligent-pricing-ceiling-configure.md (hunk 1 above)
Comment (leslietilling) [reply src/_includes/amazon-intelligent-pricing-ceiling-configure.md]: OK -- then we should leave for now. At some point we should change them for consistency.

**Comment by hickskenh** [comment]
running tests

**Pull request merged**